plugin/src/templates/edge/ipx.ts

```diff
@@ -0,0 +1,32 @@
+import { Accepts } from 'https://deno.land/x/accepts/mod.ts'
+import type { Context } from 'netlify:edge'
+
+/**
+ * Implement content negotiation for images
+ */
+
+const handler = async (req: Request, context: Context) => {
+  const { searchParams } = new URL(req.url)
+  const accept = new Accepts(req.headers)
+  const type = accept.types(['avif', 'webp'])
```

Comment:
Just AVIF and WEBP?

Reply:
Yeah. If they don't support either of those then we don't emforce a type, and let ipx serve the original format